app/configurator/components/filters.tsx

```diff
@@ -97,68 +77,41 @@ export const DimensionValuesMultiFilter = ({
       : [];
   }, [dimension?.values, locale]);
 
-  const activeFilter = dimension ? getFilterValue(state, dimension.iri) : null;
-  const isFilterActive: Set<string> = useMemo(() => {
-    if (!dimension) {
-      return new Set();
-    }
-    const activeKeys = activeFilter
-      ? activeFilter.type === "single"
-        ? [String(activeFilter.value)]
-        : activeFilter.type === "multi"
-        ? Object.keys(activeFilter.values)
-        : []
-      : [];
-    return new Set(activeKeys);
-  }, [dimension, activeFilter]);
-
-  const selectionState: SelectionState = !activeFilter
-    ? "ALL_SELECTED"
-    : isFilterActive.size === 0
-    ? "NONE_SELECTED"
-    : "SOME_SELECTED";
-
   if (data?.dataCubeByIri?.dimensionByIri) {
     return (
-      <>
-        <SelectionControls
-          dimensionIri={dimensionIri}
-          selectionState={selectionState}
-        />
-
+      <MultiFilterContextProvider dimensionData={dimension}>
+        <SelectionControls dimensionIri={dimensionIri} />
         {sortedDimensionValues.map((dv) => {
           if (state.state === "CONFIGURING_CHART") {
-            const path = colorConfigPath ? `${colorConfigPath}.` : "";
-
-            const color = get(
-              state,
-              `chartConfig.fields["${state.activeField}"].${path}colorMapping["${dv.value}"]`
-            );
-
             return (
-              <MultiFilterField
+              <Flex
                 key={dv.value}
-                dimensionIri={dimensionIri}
-                label={dv.label}
-                value={dv.value}
-                allValues={dimension?.values.map((d) => d.value) ?? []}
-                checked={
-                  selectionState === "ALL_SELECTED"
-                    ? true
-                    : selectionState === "SOME_SELECTED"
-                    ? !!isFilterActive.has(dv.value)
-                    : undefined
-                }
-                checkAction={selectionState === "NONE_SELECTED" ? "SET" : "ADD"}
-                color={color}
-                colorConfigPath={colorConfigPath}
-              />
+                sx={{
+                  justifyContent: "space-between",
+                  alignItems: "center",
+                  mb: 1,
+                  height: "2rem",
+                }}
+              >
+                <Box sx={{ maxWidth: "82%" }}>
```

Comment:
Was there a specific reason to use 82%? :D

Reply:
I'd say "no", but this is not something I change, I mostly rearranged code here ;)

Reply:
I will see what can be done about this in a future PR. @herrstucki if you remember the intention ?

Reply:
Just checked, the max-width is limited to make space for the color picker. Not sure why 82% was used though 😁

Reply:
In the next PR, I changed the code to using flex, with a flex grow item for the checkbox part and a flex shrink for the color picker.